data/obs-packaging/linux-container/README.md

```diff
@@ -0,0 +1,17 @@
+#linux-container
+
+This directory contains the sources to update rpm specfiles and debian source
+control files to generate ``cc-oci-runtime`` packages for Fedora and Ubuntu.
+
+There is a script to generate specs and dsc automatically, to be pushed to
+Open Build Service (http://openbuildservice.org/) and generate packages for you.
+
+
+``./update_kernel.sh``
+
+This scirpt will generate/update teh sources to create ``linux-container`` packages
```

Comment:
This script generates and updates the sources to create the following ``linux-container`` packages: